templates/artist/artist_detail.html

```diff
@@ -24,23 +24,26 @@ <h1 class="header-font">{{ artist.name }}</h1>
     <div class="break">
         <hr>
     </div>
-    {% thumbnail artist.photo.img "500" as thumb %}
-        <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
-    {% endthumbnail %}
+    <div class="extra-padding">
+        {% thumbnail artist.photo.img "500" as thumb %}
+            <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
+        {% endthumbnail %}
+    </div>
+    <!-- Genre -->
     {% with genres=artist.genres.all %}
     <div class="break">
```

Comment:
I just realized that there's a `{% with %}` block here. Everything inside of this block should be indented in one. Sorry if I've contradicted something else I said earlier in my review.